docs/advanced/storing-login-details.md

```diff
@@ -61,38 +87,44 @@ username=<username>
 password=<password>
 ```
 
-As with `dcrd` you can use any values you want here.  If you choose to
-use a different set of credentials than you did for `dcrd` you will
-also need to add:
+If you set the optional public password for the wallet (the second password
+during the wallet creation) you can also add that to `dcrwallet.conf`:
 
 ```no-highlight
-dcrdusername=<dcrdusername>
-dcrdpassword=<dcrdpassword>
+walletpass=<yourwalletpassword>
 ```
 
-If you use the same credentials you can leave those values out.
+You cannot specify you wallet passphrase in the config files. That
```

Comment:
I think @jrick is hoping to remove that prompt so might be best not to suggest its use here.